connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/expressions/UserDefinedFunction.scala

```diff
@@ -103,7 +103,8 @@ case class ScalarUserDefinedFunction private[sql] (
     outputType: proto.DataType,
     name: Option[String],
     override val nullable: Boolean,
-    override val deterministic: Boolean)
+    override val deterministic: Boolean,
+    aggregate: Boolean)
```

Comment:
Can you rename the class to `UserDefinedFunction`?

Reply:
How is this an issue? Are we sending it over the wire? We really shouldn't.

Reply:
Ouch my previous comment is wrong. There is a `UserDefinedFunction` class defined earlier in this file.